lib/tesla/middleware/telemetry.ex

```diff
@@ -18,6 +18,34 @@ if Code.ensure_loaded?(:telemetry) do
     end)
     ```
 
+    ## Options
+    - `:telemetry_prefix` - replaces default `[:tesla]` with desired Telemetry event prefix (see below)
```

Comment:
Since this option is passed directly to telemetry middleware I don't think we need to include the `telemetry_` prefix in the prefix name (🥁)

I'd go with just `:prefix` .

Reply:
Sounds good. Will update.

Reply:
Just updated. Let me know if that works.